coding-concepts.js

```diff
@@ -7,53 +7,55 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: this code will push the string indigo to the end of the existing array
+// b) Verify and explain:The outcome was 5, so this means that i was correct. and the code adds or "pushes" a fifth string to the array. This action gives the new length of 5 instead of 4. This is what the code would have output without the second console command.
 
 
 // --------------------1) What will this log?
 
 const cohort = "Alpha 2022"
-// console.log(cohort.length)
+ //console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This should come out with, the output of how many characters of text are in the string. This would be the "length" of the phrase in the sting it's output would be 10.
+// b) Verify and explain: I was correct, because it counts the amount of the legnth of the string. computers start counting at 0, so the 9 values in the string are counted as 10. 
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: this will run a greetin that says "Hello world!" 4 times in a row
+// b) Verify and explain: I was wrong, because it's asking for the 4th text value in the string. The output is o which is the 4th letter in the string.
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+ //console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the output should be "Ruby"
+// b) Verify and explain: I was correct the output was Ruby, becuase the index spot is 1 and since computers count from zero. "Ruby" is in the 1 spot value of the index which gives you the awnser.
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
+ //console.log(weekendDays.toUpperCase())
 
+// a) Your answer: This will print the string array in all uppercase letters
+// b) Verify and explain: I got an error saying ".toUpperCase" is not a funtion. The reason it doesn't run because there is no value selected in the index.There seems to be errors in the code, ".toUpperCase" should be used it a code more like this. --->
+const sentence = 'The quick brown fox jumps over the lazy dog.';
```

Comment:
I like that you looked up why you got this error!  The main difference between the example given and your example is that one is an array and the other is a string.  What data type does .toUpperCase() work on?